File: ClientApp/src/app/report/entity/countbymsj.ts
export class CountByMsj {

  public id !: number;
  public msj !: string;
  public count !: number;
  public perecentage !: number;

  constructor(id:number,msj:string,count:number,perecentage:number) {
    this.id=id;
    this.msj=msj;
    this.count=count;
    this.perecentage=perecentage;
  }

}

File: ServerApp/src/main/java/lk/earth/earthuniversity/report/entity/CountByMsj.java
package lk.earth.earthuniversity.report.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class CountByMsj {

    private Integer id;
    private String msj;
    private Long count;
    private double percentage;

    public CountByMsj() {  }

    public CountByMsj(String msj, Long count) {
        this.msj = msj;
        this.count = count;
    }

    public String getmsj() {
        return msj;
    }
    public void setmsj(String msj) {
        this.msj = msj;
    }
    public Long getCount() {
        return count;
    }
    public void setCount(Long count) {
        this.count = count;
    }
    public double getPercentage() {
        return percentage;
    }
    public void setPercentage(double percentage) {
        this.percentage = percentage;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Id
    public Integer getId() {
        return id;
    }

}
